docs/src/main/paradox/keyspaces.md

```diff
@@ -20,6 +20,8 @@ Configure the authentication plugin in application.conf:
 
 ```
 datastax-java-driver { 
+  profiles.akka-persistence-cassandra-profile.basic.request.consistency = LOCAL_QUORUM
+
```

Comment:
the other PR is also adding to the ssl engine ?
https://github.com/akka/akka-persistence-cassandra/pull/1029

Reply:
I just got the report about this one ⬆️ needed to be added. The additions form the other PR are not related to the consistency level, but probably also needed, I have not tried it 🤷